docs/content/sidebars/developer.js

```diff
@@ -214,6 +214,98 @@ const developer = [
         label: 'NFT',
         items: ['developer/iota-101/nft/create-nft', 'developer/iota-101/nft/rent-nft'],
     },
+    {
```

Comment:
Please move this after the dev cheat sheet, it kind of gets lost where it is now

Reply:
Once it's available on the testnet we could consider giving the docs their own section, as the ts sdk

Reply:
@lucas-tortora It is available on testnet

Reply:
Let me put something together real quick

Reply:
https://github.com/iotaledger/iota/pull/4381